### PR TITLE
Add additional ns info.  

### PR DESCRIPTION
**Pull request by RickJWagner**
Add additional ns info.  Formatted so gofmt should pass.

### CODE REVIEW

File: pkg/cmd/inspect/resource.go [hunk 1]
```diff
@@ -179,8 +179,11 @@ func (o *InspectOptions) gatherNamespaceData(baseDir, namespace string) error {
 	}
 
 	resourcesTypesToStore := map[schema.GroupVersionResource]bool{
-		corev1.SchemeGroupVersion.WithResource("events"):     true,
-		corev1.SchemeGroupVersion.WithResource("configmaps"): true,
+		corev1.SchemeGroupVersion.WithResource("events"):          true,
+		corev1.SchemeGroupVersion.WithResource("configmaps"):      true,
+		corev1.SchemeGroupVersion.WithResource("endpoints"):       true,
+		corev1.SchemeGroupVersion.WithResource("podtemplates"):    true,
```
Comment (soltysh) [inline pkg/cmd/inspect/resource.go]: These are not actual resources, you might want to have just pods.

**Comment by deads2k** [comment]
Have a look to see what isn't already covered by `all` and you add it here https://github.com/openshift/must-gather/blob/master/pkg/cmd/inspect/namespace.go#L18

**Comment by RickJWagner** [comment]
This PR is no longer relevant.   ResourcesTypesToStore now lives in namespace.go and contains only 'pods', as @soltysh suggested in February.

**Pull request closed without merging**